File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    /**
    public UberVan(String license, Account driver, Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
    */

    public UberVan(String license, Account driver) {
        super(license, driver);
    }

    @Override
    public void setPassengers(Integer passengers) {
        if (passengers == 6) {
            this.passengers = passengers;
        } else {
            System.out.println("You need assign 6 passengers");
        }
    }

}

File: PHP/UberVan.php
<?php
require_once("Car.php");

class UberVan extends Car {
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial) {
        parent::__construct($license, $driver);
        $this->typeCarAcepcted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }

    public function setPassengers($passengers) {
        if ($passengers == 6) {
            $this->passengers = $passengers;
        } else {
            echo "You need assign 6 passengers";
        }
    }
    
}
?>
